molecule/common/tests/test_cluster.py

```diff
@@ -13,8 +13,10 @@
 )
 testinfra_hosts = ansible_runner.get_hosts('all')
 
+scenario_name = os.environ['MOLECULE_SCENARIO_NAME']
```

Comment:
```suggestion
scenario_name = os.environ.get('MOLECULE_SCENARIO_NAME', 'default')
```

The same comment for others

Reply:
This variable is added by `molecule` - it's always defined.